tests/queries/0_stateless/02319_quantile_interpolated_weighted.sql

```diff
@@ -24,4 +24,17 @@ SELECT quantilesInterpolatedWeighted(0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8
 SELECT quantilesInterpolatedWeighted(0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0)(b, 2) FROM decimal;
 SELECT quantilesInterpolatedWeighted(0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0)(c, 3) FROM decimal;
 
+SELECT 'quantileExactWeightedInterpolated';
```

Comment:
Please create a new file. It's much better for the fuzzer (it will pick it up and iterate over it). Also please add some tests with disk storage (-Merge with a MV or just insert into a -State column) and the different supported types

Reply:
This is still relevant. Any aggregation should test the -State and -Merge process via MVs